core/src/main/java/org/arquillian/smart/testing/hub/storage/local/DuringExecutionLocalStorage.java

```diff
@@ -53,7 +57,7 @@ private void storeDirectory(String dirNameToStore, String targetDir) {
             Path dirToStore = new LocalStorage(rootDir)
                 .afterExecution(targetDir)
                 .getPathTo(dirNameToStore);
-
+            logger.debug("copying directories from %s to %s", dirToCopy, dirToStore);
```

Comment:
Is it necessary? Imagine what it will look like in multi-module project... A lot of same lines that don't contain any additional value... Or is there any use case I don't see?

Reply:
While storing modified pom.xml we are logging that it'll be stored at `.smart-testing/reporting/` directory. But at the end of execution we are moving it `target` folder. But then if there is no logs about this. Then user will try to find modified pom.xml in `.smart-testing` which won't be there at the end of execution. 

So it's quite difficult for user to identify where is it?

So I had two options:
Either we should not log where modified pom.xml is stored initially or log where is it stored & moved

Reply:
Yeah, now I probably understand why you did this. You store the effective pom file before the test execution, then (after test execution) the dir is moved to target, so you probably wanted to show where the effective pom can be found... To be honest, I wouldn't associate these two lines so I would be still trying to find the effective pom inside of the original directory - you know, I see a [line](https://github.com/arquillian/smart-testing/pull/225/files#diff-1848634e0d5e47585ae03dfabcf1e835R36) saying that the effective pom is stored somewhere, then I would expect that it is still there.
That is the reason why I didn't want to store the effective pom files before test execution, but after...

Reply:
maybe we can give info like "effective pom.xml stored in `.smart-testing` for temporary but you can find it in `target/smart-testing` at the end of execution

Reply:
but look we are not traversing again for all modules if we did it duringExecution phase which will save few seconds for big multi-module project

Reply:
wouldn't solve all the problems just storing it once in `afterSessionEnd` phase?

Reply:
but again we need to traverse all projects once in `afterSessionEnd` phase